### PR TITLE
Add initial project structure

### PR DESCRIPTION
**Pull request by ayeshLK**
Part of: [#6864](https://github.com/ballerina-platform/ballerina-library/issues/6864)

### CODE REVIEW

File: .github/workflows/build-with-bal-test-graalvm.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+concurrency:
+  group: graalvm-check
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'Skip GraalVM Check')
+
+    steps:
```
Comment (Nuvindu) [inline .github/workflows/build-with-bal-test-graalvm.yml:19]: Can't we use the GraalVM connector template for this workflow file?

File: ballerina/build.gradle [hunk 2]
```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org)
```
Comment (Nuvindu) [inline ballerina/build.gradle:2]:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org)
```

File: native/src/main/java/module-info.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org)
```
Comment (Nuvindu) [inline native/src/main/java/module-info.java:2]:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
```

File: settings.gradle [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    id "com.gradle.enterprise" version "3.13.2"
+}
+
+def projectName = 'aws.marketplace.mpe'
+rootProject.name = "module-ballerinax-${projectName}"
+
+include ":checkstyle"
+include ":${projectName}-native"
+include ":${projectName}-ballerina"
+//include ":${projectName}-examples"
```
Comment (Nuvindu) [inline settings.gradle:50]: If this is not necessary now, we can remove the comment

File: .github/workflows/trivy-scan.yml [hunk 5]
```diff
@@ -0,0 +1,35 @@
+name: Trivy
+
+on:
+  workflow_dispatch:
+
+jobs:
+  ubuntu-build:
```
Comment (Nuvindu) [inline .github/workflows/trivy-scan.yml:7]: Can't we use this template here?
https://github.com/ballerina-platform/ballerina-library/blob/main/library-templates/generated-connector-template/files/.github/workflows/trivy-scan.yml

File: native/src/main/java/module-info.java (hunk 3 above)
Comment (Nuvindu) [reply native/src/main/java/module-info.java:2]: check for other places

File: .github/workflows/build-with-bal-test-graalvm.yml (hunk 1 above)
Comment (ayeshLK) [reply .github/workflows/build-with-bal-test-graalvm.yml]: Fixed with [69492b9e569069b5ab740b394085504a601f2728](https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpe/pull/1/commits/69492b9e569069b5ab740b394085504a601f2728)

File: .github/workflows/trivy-scan.yml (hunk 5 above)
Comment (ayeshLK) [reply .github/workflows/trivy-scan.yml]: Fixed with [69492b9e569069b5ab740b394085504a601f2728](https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpe/pull/1/commits/69492b9e569069b5ab740b394085504a601f2728)

File: settings.gradle (hunk 4 above)
Comment (ayeshLK) [reply settings.gradle:50]: Lets keep this as it is. We will be using in the future.

File: ballerina/build.gradle (hunk 2 above)
Comment (ayeshLK) [reply ballerina/build.gradle]: Fixed with [019b0785f86e82425a6cab4f91f1e4cb5e417aa0](https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpe/pull/1/commits/019b0785f86e82425a6cab4f91f1e4cb5e417aa0)

File: .gitignore
Comment (NipunaRanasinghe) [inline .gitignore:1]: It seems some of the entries we've added for the other connectors are missing in here. Shall we check and include the missing items from https://github.com/ballerina-platform/ballerina-library/blob/main/library-templates/generated-connector-template/files/.gitignore

Comment (ayeshLK) [reply .gitignore:1]: I referred to the mongodb connector for the git-ignore configurations [1] Updated the gitignore with required configs with [5de3c0b7dc24a58c897b627d7addb452bdc1955c](https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpe/pull/1/commits/5de3c0b7dc24a58c897b627d7addb452bdc1955c)

[1] - https://github.com/ballerina-platform/module-ballerinax-mongodb/blob/master/.gitignore

File: .gitignore [hunk 6]
```diff
@@ -13,12 +13,40 @@
...
+.m2
+
+# Ignore output directories
+build
+target
+generated
+
+# Environment files
+*.env
+Config.toml
```
Comment (NipunaRanasinghe) [inline .gitignore:51]:
```suggestion
**/Config.toml
```

File: build-config/resources/Ballerina.toml [hunk 7]
```diff
@@ -0,0 +1,19 @@
+[package]
+distribution = "2201.9.2"
+org = "ballerinax"
+name = "aws.marketplace.mpe"
+version = "@toml.version@"
+license= ["Apache-2.0"]
+authors = ["Ballerina"]
+keywords = ["IT Operations/Databases", "Cost/Freemium"]
```
Comment (NipunaRanasinghe) [inline build-config/resources/Ballerina.toml:8]: Are these keywords correct? 
```suggestion
keywords = ["IT Operations/Databases", "Cost/Freemium"]
```

Comment (ayeshLK) [reply build-config/resources/Ballerina.toml]: Fixed with [4a8455523de49bd042c328efd95710ab28b77f8a](https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpe/pull/1/commits/4a8455523de49bd042c328efd95710ab28b77f8a)

**Review by NipunaRanasinghe — DISMISSED** [review]

**Review by DimuthuMadushan — APPROVED** [review]

**Pull request merged**